docs/content/doc/packages/cran.zh-cn.md

```diff
@@ -90,4 +90,4 @@ install.packages("{package_name}")
 
 ```shell
 install.packages("testpackage")
-```
\ No newline at end of file
+```
```

Comment:
?

Reply:
it's a new line at the end of the file change

Reply:
markdownlint requires files to end in a single newline